bundle/Form/DataWrapper.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Bundle\InformationCollectionBundle\Form;
+
+final class DataWrapper
+{
+    /**
+     * One of the Ibexa Platform structs, like ContentCreateStruct, UserUpdateStruct and so on.
+     *
+     * @var mixed
+     */
+    public $payload;
+
+    /**
+     * Definition of the target.
+     *
+     * In case of Content or User target, this must be the corresponding ContentType.
+     *
+     * @var mixed|null
+     */
+    public $definition;
+
+    /**
+     * The target struct that applies to. E.g. Content, User, Section object and so on.
```

Comment:
Adapt descriptions for info collection context.

Reply:
solved in 4ccfd43d